frontend/src/components/donation_page/index.css

```diff
@@ -0,0 +1,16 @@
+.content-styling {
+  font-size: 32px;
+  margin-left: 266px;
+  margin-right: 266px;
```

Comment:
can we add some styling for mobile? I do not think we want these margins at a lower a screen size.

Reply:
Done. I left the 266px for the large screen size as per the mock up directions.